### DISCUSSION TITLE
Eddie Not updating  to Edsm

### DISCUSSION THREAD

**Discussion by emyrsmorgan** [EDCD/EDDI#2536] [EDDI VoiceAttack Q&A]
For some reason Eddie has stopped updating to edsm. I even tried generating a new key and recopying it from Edsm with no luck

**Comment by Tkael** [comment]
What version of EDDI are you using?
Have you verified that your EDSM commander name matches the name configured in EDDI?
Have you reviewed EDDI's log files (located at `%appdata%/EDDI`) to see if any exceptions have been recorded?

**Reply by Tkael** [reply]
And are you playing on the legacy server or the Horizons 4.0 / Odyssey live galaxy server?

**Reply by emyrsmorgan** [reply]
Oddysey Live eddie version 4.0.2 Voice Attack Version 1.10.3 Cmdr Thandco it matches with edsm
2023-05-03T21:04:12 [Warning] StarMapService:SetEdsmCredentials EDSM Responder not configured: Commander name not set.
2023-05-03T21:04:12 [Info] StarSystemSqLiteRepository:CreateOrUpdateDatabase Starsystem repository schema is version 3
2023-05-03T21:04:12 [Info] EDDNResponder:.ctor Initialized EDDN responder
2023-05-03T21:04:12 [Warning] StarMapService:SetEdsmCredentials EDSM Responder not configured: Commander name not set.
 IN Eddie it shows my correct name   Thandco  under commander details

**Reply by emyrsmorgan** [reply]
there alot of other errors though

[eddi.log](https://github.com/EDCD/EDDI/files/11391344/eddi.log)

**Comment by emyrsmorgan** [comment]
i tried obtain Edsm log and that works

**Reply by Tkael** [reply]
Interesting. That would suggest a temporary problem - perhaps with accessing your config files? But if you were able to obtain your EDSM logs then the EDSM credentials must have righted themselves.

**Comment by emyrsmorgan** [comment]
i reinstalled eddie and voice attack seemed to fix it the problem was while it could Pull the logs  From Edsm it was not exporting to Edsm wierd

**Comment by Tkael** [comment]
I understood this as resolved so I'm going to ahead and close it. Please let me know if otherwise. o7
